bin/cron/ops_data_pull

```diff
@@ -42,23 +35,19 @@ SeamlessDatabasePool.use_persistent_read_connection do
         :District => user.district.try(:name) || 'None',
         :School => user.ops_school || 'None'
       }
-      level_progress = summarize_user_progress(script, user)[:levels]
-      percent = percent_complete(script, user)
-      script.stages.find_each do |stage|
-        level_hash = {}
-        stage.script_levels.map(&:level).each do |level|
-          progress = level_progress[level.id]
-          level_hash[level.id.to_s] = %w(perfect passed).include?(progress.try(:[], :status)) ? '1' : '0'
-        end
-        row.merge!(level_hash)
-        row.merge!({:"Stage #{stage.absolute_position} Percent Complete" => percent[stage.absolute_position - 1].to_s})
-      end
-      row[:'Script Percent Complete'] = percent_complete_total(script, user).to_s
+      row[:'Script Percent Complete'] = (
+        100.0 * percent_complete_total(script, user)
```

Comment:
Should we rename the function since it's not actually returning a percent? Maybe something like `calculate_completion_ratio`?